travis/cfg/travis_run_pylint_beta.cfg

```diff
@@ -1,6 +1,6 @@
 [MASTER]
 profile=no
-ignore=CVS,.git,scenarios,.bzr
+ignore=CVS,.git,scenarios,.bzr,account_parent,web_gantt_native,web_gantt_project_native
```

Comment:
you're missing html_tree_widget, web_hide_db_manager_link, web_widget_colorpicker and web_widget_time_delta